velox/exec/tests/WindowTest.cpp

```diff
@@ -79,6 +79,74 @@ TEST_F(WindowTest, spill) {
   ASSERT_GT(stats.spilledPartitions, 0);
 }
 
+TEST_F(WindowTest, rankLikeWithEqualValue) {
```

Comment:
Just leaving a comment here to ensure these code-paths are covered by the WindowFuzzer. Please take a look at it to verify that it generates plans that use RowStreamingWindowBuild. Also that it generates such plans with a good frequency.

Reply:
@aditi-pandit All StreamingWindow tests, if they meet the conditions, can reach this part of the code. I see that the WindowFuzzer includes the addition of StreamingWindow tests [here](https://github.com/facebookincubator/velox/blob/main/velox/exec/fuzzer/WindowFuzzer.cpp#L313), so it can cover  these code-paths.